tests/consensus_tests/test_consensus_lag.py

```
@@ -0,0 +1,116 @@
+Every peer keeps a ring of the entries it applied, stamped with its own clock. The
+endpoint collects those rings from the whole cluster and measures each entry from
+whichever peer applied it first, so a lag is never negative.
+
+Needs `--features staging` for `test_slow_down`.
+"""
+
+import json
+import pathlib
+
+from .fixtures import create_collection, upsert_random_points
+from .utils import *
```

Ruff (0.16.1)

[error] `from .utils import *` used; unable to detect undefined names

(F403)